src/LiveSplit.Core/Server/CommandServer.cs

```diff
@@ -291,6 +291,96 @@ private void ProcessMessage(string message, IConnection clientConnection)
+                bool doSimplePlatform = !State.Run.Metadata.GameAvailable;
+                if (!string.IsNullOrEmpty(State.Run.Metadata.PlatformName) && (doSimplePlatform || (State.Run.Metadata.Game != null && State.Run.Metadata.Game.Platforms.Count > 1)))
+                {
+                    if (State.Run.Metadata.UsesEmulator)
+                    {
+                        keyL.Add($"\"platform\": \"{State.Run.Metadata.PlatformName} Emulator\"");
+                    }
+                    else
+                    {
+                        keyL.Add($"\"platform\": \"{State.Run.Metadata.PlatformName}\"");
+                    }
+                }
+                else if (State.Run.Metadata.UsesEmulator)
+                {
+                    keyL.Add($"\"platform\": \"Emulator\"");
+                }
+                //Variables
+                var vKeys = new List<string>();
+                IEnumerable<string> variables = State.Run.Metadata.VariableValueNames.Keys;
+                if (State.Run.Metadata.GameAvailable && State.Run.Metadata.Game != null)
+                {
+                    string categoryId = null;
+                    if (State.Run.Metadata.CategoryAvailable && State.Run.Metadata.Category != null)
+                    {
+                        categoryId = State.Run.Metadata.Category.ID;
+                    }
+                    variables = State.Run.Metadata.Game.FullGameVariables.Where(x => x.CategoryID == null || x.CategoryID == categoryId).Select(x => x.Name);
+                }
+                foreach (string variable in variables)
+                {
+                    if (State.Run.Metadata.VariableValueNames.ContainsKey(variable))
+                    {
+                        string name = variable.TrimEnd('?');
+                        string variableValue = State.Run.Metadata.VariableValueNames[variable];
+                        string valueLower = variableValue.ToLowerInvariant();
+                        if (valueLower == "yes")
+                        {
+                            vKeys.Add($"\"{variable}\": \"{name}\"");
+                        }
+                        else if (valueLower == "no")
+                        {
+                            vKeys.Add($"\"{variable}\": \"No {name}\"");
+                        }
+                        else
+                        {
+                            vKeys.Add($"\"{variable}\": \"{variableValue}\"");
+                        }
+                    }
+                }
+                if (vKeys.Any())
+                {
+                    String variablePairs = $"{{{string.Join(", ", vKeys)}}}";
+                    keyL.Add($"\"variables\": {variablePairs}");
+                }
+                //Combine
+                String keys = "";
+                if (keyL.Any())
+                {
+                    keys = string.Join(", ", keyL);
+                }
+                response = $"{{{keys}}}";
```

Comment:
```suggestion
                response = "{" + string.Join(", ", keyL) + "}";
```
This should really be done with a `StringBuilder`.